backend/RoomPlanner/room-planner/src/test/java/edu/roomplanner/controller/UsersRestControllerTest.java

```diff
@@ -38,62 +42,60 @@
     @Autowired
     private Flyway flyway;
 
+    private RequestPostProcessor bearerToken;
+
     @Before
     public void init() {
         flyway.clean();
         flyway.migrate();
+
+        bearerToken = oAuthHelper.addBearerToken("sghitun@yahoo.com", "person");
     }
 
     @Autowired
     private UserRepository userRepository;
 
+    @Autowired
+    OAuthHelper oAuthHelper;
+
     @Test
     public void shouldReturnResponseEntityWithValidRoomDtoListAndStatusFoundWhenGetAllRoomsIsCalled() throws Exception {
 
-        RoomDto roomDtoOne = BuildersWrapper.buildRoomDto(1L, "Wonderland", 5, 14);
-        RoomDto roomDtoTwo = BuildersWrapper.buildRoomDto(2L, "Westeros", 8, 21);
-        List<RoomDto> roomDtoList = Arrays.asList(roomDtoOne, roomDtoTwo);
+        RoomDto roomDtoOne = BuildersWrapper.buildRoomDto(2L, "wonderland@yahoo.com", "Wonderland", null, 5, 14, UserType.ROOM);
```

Comment:
Where is you setup for this method?
Are you expecting entities created from scrips?